code_demo/Filter.ipynb

```diff
@@ -182,31 +182,19 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 10,
+   "execution_count": 12,
    "metadata": {},
    "outputs": [
     {
      "name": "stdout",
      "output_type": "stream",
      "text": [
-      "[====================] 100.000000%"
+      "Calculation time = 0.02mins\n"
      ]
     }
    ],
    "source": [
-    "finalfinal = []\n",
-    "for j in chispace:\n",
-    "    #A silly way to parallelize...\n",
-    "    final = Parallel(-1)(delayed(fit.likelihood_vs_mass_spin)(i, j, **input) \n",
-    "                         for i in massspace)\n",
-    "    finalfinal.append(final)\n",
-    "\n",
-    "    #print the process\n",
-    "    sys.stdout.write('\\r')\n",
-    "    sys.stdout.write(\"[%-20s] %f%%\" % ('='*int(20*j/chispace[-1]), 100*j/chispace[-1]))\n",
-    "    sys.stdout.flush()\n",
-    "    \n",
-    "finalfinal = np.array(finalfinal)"
+    "finalfinal = qnm_filter.parallel_compute(fit.likelihood_vs_mass_spin, massspace, chispace, kwargs=input)"
```

Comment:
Remove `import sys` and `from joblib import Parallel, delayed` when they are not needed.